grails-app/domain/org/pih/warehouse/inventory/CycleCountDetails.groovy

```diff
@@ -32,7 +34,7 @@ class CycleCountDetails implements Serializable {
     String blindCountVarianceComment
 
     // Verification count details
-    Date verificationCountDateCounted
+    Instant verificationCountDateCounted
```

Comment:
- dateInitiated:  from cycleCount.dateCreated        (which is an Instant)
- dateRecorded: from transaction.transactionDate   (which is a Date but we can use Instant because this is a view and so is populated via sql query and not via interacting with the domain)
- dateRequested: from cycleCountRequest.dateCreated (which is an Instant)
- blindCountDateCounted: from CycleCountItem.dateCounted (which is an Instant)
- verificationCountDateCounted: from CycleCountItem.dateCounted (which is an Instant)